databridge.camel-sql/src/main/resources/sqlconsumer.schema.json

```diff
@@ -0,0 +1,53 @@
+{
+	"$id": "https://example.com/person.schema.json",
+	"$schema": "https://json-schema.org/draft/2020-12/schema",
+	"title": "mqttconsumer.json",
+	"type": "object",
+	"required": [
+		"uniqueId",
+		"serverUrl",
+		"db",
+		"query"
+	],
+	"properties": {
+		"uniqueId": {
+			"type": "string",
+			"description": "Unique ID to match options",
+			"pattern": "^[a-z0-9-]+$"
+		},
+		"serverUrl": {
+			"type": "string",
+			"description": "IP or Hostname of the SQL Server",
+            "default": "127.0.0.1",
+            "pattern": "^([a-z0-9-_\\.]+\\.[a-z]{1,3}|[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3})$"
+		},
+		"serverPort": {
+			"type": "integer",
+			"description": "Port of the MQTT Broker",
+            "minimum": 0,
+            "maximum": 65535
+		},
+		"db": {
+			"type": "string",
+			"description": "Database vendor",
+			"pattern": "^(mariadb|postgresql|sqlite)$"
+		},
+		"dbName": {
+			"type": "string",
+			"description": "Database name"
+		},
+		"user": {
+			"type": "string",
+			"description": "Database user name"
+		},
+		"password": {
+			"type": "string",
+			"description": "Database user password"
+		},
+		"query": {
+			"type": "string",
+			"description": "SQL query to execute",
+			"pattern": "^(?!.*(CREATE|DELETE|UPDATE|DROP)).*$"
```

Comment:
please replace the regex here with: `^(?!.*(?:[cC][rR][eE][aA][tT][eE]|[dD][eE][lL][eE][tT][eE]|[uU][pP][dD][aA][tT][eE]|[dD][rR][oO][pP])).*$`  to make sure no variation of the words "CREATE", "DELETE", "UPDATE", "DROP" is allowed.

Reply:
actually we have to reconsider this regex because it would not let you query for entries that e.g. contain one of the disallowed words...